virttest/nfs.py

```diff
@@ -327,9 +327,13 @@ def setup_remote(self):
 
         self.mount_src = "%s:%s" % (self.nfs_server_ip, self.mount_src)
         logging.debug("Mount %s to %s" % (self.mount_src, self.mount_dir))
-        mount_cmd = ssh_cmd + "'mount -t nfs %s %s -o %s'" % (self.mount_src,
-                                                              self.mount_dir,
-                                                              self.mount_options)
+        if self.mount_options:
+            mount_cmd = ssh_cmd + "'mount -t nfs %s %s -o %s'" % (self.mount_src,
+                                                                  self.mount_dir,
+                                                                  self.mount_options)
+        else:
+            mount_cmd = ssh_cmd + "'mount -t nfs %s %s'" % (self.mount_src,
+                                                            self.mount_dir)
```

Comment:
Yes, good catch. I'd suggest to take the opportunity to improve the code as well, removing the duplications and improving the readability. Something like:
```python
        mount_cmd = "mount -t nfs %s %s" % (self.mount_src, self.mount_dir)
        if self.mount_options:
            mount_cmd += " -o %s" % self.mount_options
        try:
            cmd = "%s '%s'" % (ssh_cmd, mount_cmd)
            process.system(mount_cmd, verbose=True)
        except process.CmdError:
            raise exceptions.TestFail("Failed to run: %s" % cmd)
```